app/models/mixins/supports_feature_mixin.rb

```diff
@@ -58,24 +58,26 @@ module SupportsFeatureMixin
   extend ActiveSupport::Concern
 
   QUERYABLE_FEATURES = {
-    :associate_floating_ip    => 'Associate a Floating IP',
-    :control                  => 'Basic control operations', # FIXME: this is just a internal helper and should be refactored
-    :cloud_tenant_mapping     => 'CloudTenant mapping',
-    :cinder_service           => 'Cinder storage service',
-    :delete                   => 'Deletion',
-    :disassociate_floating_ip => 'Disassociate a Floating IP',
-    :discovery                => 'Discovery of Managers for a Provider',
-    :evacuate                 => 'Evacuation',
-    :events                   => 'Query for events',
-    :launch_cockpit           => 'Launch Cockpit UI',
-    :live_migrate             => 'Live Migration',
-    :migrate                  => 'Migration',
-    :provisioning             => 'Provisioning',
-    :reconfigure              => 'Reconfiguration',
-    :regions                  => 'Regions of a Provider',
-    :resize                   => 'Resizing',
-    :retire                   => 'Retirement',
-    :smartstate_analysis      => 'Smartstate Analaysis',
+    :associate_floating_ip        => 'Associate a Floating IP',
+    :control                      => 'Basic control operations', # FIXME: this is just a internal helper and should be refactored
+    :cloud_tenant_mapping         => 'CloudTenant mapping',
+    :cloud_volume_backup_create   => 'CloudVolume backup creation',
+    :cloud_volume_backup_restore  => 'CloudVolume backup restore',
+    :cinder_service               => 'Cinder storage service',
```

Comment:
@durandom When the global registry of features was introduced, I didn't like it for some reason, but I couldn't express why.  I think this is part of the reason.  Note here is a "cinder" service that is at the global feature level.  This breaks the pluggability aspect.  Then again, perhaps this just should be a generic feature?  I'm not even sure why the UI would need this feature declared in the first place.

Anyway, @durandom, thoughts?